sklearn/utils/isfinite.pyx

```diff
@@ -0,0 +1,76 @@
+# cython: profile=True, linetrace=True
+# Author: jakirkham
+
+cimport libc
+cimport libc.math
+from libc.math cimport isnan as _isnan, isinf as _isinf
+
+cimport cython
+from cython.parallel import prange
+cimport numpy
+import numpy
+numpy.import_array()
+
+
+cdef fused fprecision:
+    numpy.npy_float32
+    numpy.npy_float64
+
+cdef fused const_fprecision:
+    const numpy.npy_float32
+    const numpy.npy_float64
+
+@cython.cdivision(True)
```

Comment:
This is always true

```suggestion
```

https://github.com/scikit-learn/scikit-learn/blob/effdd6e215c67f2ae8ed1e378ea1661e936059a4/sklearn/_build_utils/__init__.py#L86